docker-compose.katana.yaml

```diff
@@ -1,23 +1,23 @@
-version: "3.2"
+version: '3.2'
 
 services:
   starknet:
-    image: ghcr.io/dojoengine/dojo:latest
+    image: ghcr.io/dojoengine/dojo:nightly-5cbe549ec0d8c17fe1abbddea36bf55228043107
```

Comment:
pin dojo as the latest changed from `--allow-zero-max-fee` to `--disable-fee`, will create an issue for this